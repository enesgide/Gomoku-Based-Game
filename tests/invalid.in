who 
who
view 
view
place 
place A1
place A1
place A2
Place A1
place A21
place B123
history 
history
view
term 
 term
resign 
resign