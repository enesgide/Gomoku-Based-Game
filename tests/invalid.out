Invalid!
B
Invalid!
J10,.................................................
Invalid!
Occupied coordinate
Invalid!
Invalid coordinate
Invalid coordinate
Invalid!
A1A2
M17,xxxxxxx..........................................
Invalid!
Invalid!
Invalid!
White wins!
A1A2
Thank you for playing!
